src/panels/config/voice-assistants/debug/assist-render-pipeline-run.ts

```diff
@@ -71,13 +71,15 @@ const maybeRenderError = (
 const renderProgress = (
   hass: HomeAssistant,
   pipelineRun: PipelineRun,
-  stage: PipelineRun["stage"]
+  stage: PipelineRun["stage"],
+  start_suffix: string = "-start",
+  end_suffix: string = "-end"
```

Comment:
Why add a feature that is not going to be used?

Reply:
It felt wrong to not complete the pair I guess :smile:
Removed it.